doc/source/how-to/documenting.rst

```diff
@@ -464,8 +464,8 @@ The resulting PDF and intermediate LaTeX files are created in the
 
 Enabling multi-version documentation
 ------------------------------------
-With the release of `pyansys/actions@v3
-<https://actions.docs.pyansys.com/release/2.0/index.html>`_, projects can
+With the release of `pyansys/actions@v4
+<https://actions.docs.pyansys.com/release/4.0/index.html>`_, projects can
```

Comment:
```suggestion
<https://actions.docs.pyansys.com/version/stable/index.html>`_, projects can
```
isn't the new docs are going to be in this layout?

Reply:
You are right, @Revathyvenugopal162. Thanks for adding this.